micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/QuickStart/QuickSetup.js

```
@@ -0,0 +1,91 @@
+    },
+  ];
+  return (
+    <Card className={"sandbox-guide"} style={{ width: "25rem", height: "47rem", overflowY: "scroll" }}>
+      {/* <Card style={{ height: "47rem", overflowY: "scroll" }}> */}
+      <CardHeader>Guide To Sandbox</CardHeader>
+      <div style={{ display: "flex", flexWrap: "wrap" }}>
+        {cardConfig.map((config, index) => {
+          return config?.type === "faqs" ? (
+            <FAQ key={config.id} title={config.title} content={config.content} faqs={config.actions} />
+          ) : (
+            <React.Fragment>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)